systest/Makefile

```diff
@@ -7,7 +7,6 @@ clusters ?= 1
 size ?= 10
 level ?= debug
 bootstrap ?= 5m
-node_selector ?= cloud.google.com/gke-nodepool=dmitry-ci-preemtible-1
```

Comment:
i meant to remove `cloud.google.com/gke-nodepool=dmitry-ci-preemtible-1` but keep everything else. so by default node_selector will be empty, and will be overwritten only when tests are running in the cloud

Reply:
done